tieredstore/src/main/java/org/apache/rocketmq/store/tiered/metrics/TieredStoreMetricsManager.java

```diff
@@ -0,0 +1,323 @@
+
+public class TieredStoreMetricsManager {
+    private static final Logger logger = LoggerFactory.getLogger(TieredStoreUtil.TIERED_STORE_LOGGER_NAME);
+    public static Supplier<AttributesBuilder> attributesBuilderSupplier;
+
+    public static LongHistogram apiLatency = new NopLongHistogram();
+
+    // tiered store provider metrics
+    public static LongHistogram providerRpcLatency = new NopLongHistogram();
+    public static LongHistogram uploadBytes = new NopLongHistogram();
+    public static LongHistogram downloadBytes = new NopLongHistogram();
+
+    public static ObservableLongGauge dispatchBehind = new NopObservableLongGauge();
+    public static ObservableLongGauge dispatchLatency = new NopObservableLongGauge();
+    public static LongCounter messagesDispatchTotal = new NopLongCounter();
+    public static LongCounter messagesOutTotal = new NopLongCounter();
+    public static LongCounter fallbackTotal = new NopLongCounter();
+
+    public static ObservableLongGauge cacheCount = new NopObservableLongGauge();
+    public static ObservableLongGauge cacheBytes = new NopObservableLongGauge();
+    public static LongCounter cacheAccess = new NopLongCounter();
+    public static LongCounter cacheHit = new NopLongCounter();
+
+    public static ObservableLongGauge storageSize = new NopObservableLongGauge();
+    public static ObservableLongGauge storageMessageReserveTime = new NopObservableLongGauge();
+
+    public static List<Pair<InstrumentSelector, View>> getMetricsView() {
+        ArrayList<Pair<InstrumentSelector, View>> res = new ArrayList<>();
+
+        InstrumentSelector providerRpcLatencySelector = InstrumentSelector.builder()
+            .setType(InstrumentType.HISTOGRAM)
+            .setName(HISTOGRAM_PROVIDER_RPC_LATENCY)
+            .build();
+
+        InstrumentSelector rpcLatencySelector = InstrumentSelector.builder()
+            .setType(InstrumentType.HISTOGRAM)
+            .setName(HISTOGRAM_API_LATENCY)
+            .build();
+
+        View rpcLatencyView = View.builder()
+            .setAggregation(Aggregation.explicitBucketHistogram(Arrays.asList(1d, 10d, 100d, 200d, 400d, 600d, 800d, 1d * 1000, 1d * 1500, 1d * 3000)))
+            .setDescription("tiered_store_rpc_latency_view")
+            .build();
+
+        InstrumentSelector uploadBufferSizeSelector = InstrumentSelector.builder()
+            .setType(InstrumentType.HISTOGRAM)
+            .setName(HISTOGRAM_UPLOAD_BYTES)
+            .build();
+
+        InstrumentSelector downloadBufferSizeSelector = InstrumentSelector.builder()
+            .setType(InstrumentType.HISTOGRAM)
+            .setName(HISTOGRAM_DOWNLOAD_BYTES)
+            .build();
+
+        View bufferSizeView = View.builder()
+            .setAggregation(Aggregation.explicitBucketHistogram(Arrays.asList(1024d, 10d * 1024, 100d * 1024, 1024d * 1024, 10d * 1024 * 1024, 32d * 1024 * 1024, 50d * 1024 * 1024, 100d * 1024 * 1024)))
```

Comment:
Here can we use     
private static final double KB = 1024d;
1 * KB, 10 * KB ....  
to improve the readability?